app/controllers/comment_controller.php

```diff
@@ -0,0 +1,58 @@
+<?php
+class CommentController extends AppController
+{
+    function __construct($name)
```

Comment:
I would recommend to add this in the `AppController` inside `beforeFilter()` method since you would have to do this in every controller to check if user is logged in
Also I think it's not safe to modify the parent construct in the controller :sweat: